lib/sse_test_server/sse_handler.ex

```diff
@@ -0,0 +1,42 @@
+defmodule SSETestServer.SSEHandler do
+
+  alias SSETestServer.SSEServer
+
+  defmodule State do
+    @enforce_keys [:path, :sse_server]
+    defstruct path: nil, sse_server: nil, delay: nil
+  end
+
+  # TODO: Find a way to get rid of the chunk wrappers around all this stuff to
+  # allow testing clients that don't handle transport-encodings transparently.
```

Comment:
Being able to enable/disable chunked encoding would be good, I think.

Reply:
Yup. Unfortunately this is kind of hardcoded in the bowels of `cowboy`. It may be possible to write a "stream handler" (which I don't completely understand yet) to modify the response on the fly and strip out the chunking stuff, but that's a problem for another day.